### PR TITLE
Update requirements.md

### PR DESCRIPTION
**Pull request by scnbwy**
Office 365 operated by 21Vianet is not supported

### CODE REVIEW

**Comment by scnbwy** [comment]
#2934

**Comment by scnbwy** [comment]
Sign in to Microsoft Teams Rooms (Skype for Business Mode only) using the 21Vianet Office365 resource account to indicate that username password verification has failed, but the same account can be logged in on Client.

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by scnbwy** [comment]
> @nenonix @ JohanFreelancer9此PR需要进行复制编辑。谢谢。

Hi，scanum
Skype for business online used in all of MTR's descriptions without listing China 21Vianet accounts are not supported，So too much update is required in the Docs MTR section

File: Teams/room-systems/requirements.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Also, refer to:
 - [License options based on your plan: Microsoft Teams Rooms](/SkypeForBusiness/skype-for-business-and-microsoft-teams-add-on-licensing/license-options-based-on-your-plan/skype-room-systems-v2)
 
 > [!NOTE]
-> Microsoft Teams Rooms is intended for use with Microsoft Teams, Skype for Business Server 2019, Skype for Business Server 2015, or Skype for Business Online.
+> Microsoft Teams Rooms is intended for use with Microsoft Teams, Skype for Business Server 2019, Skype for Business Server 2015, or Skype for Business Online（Office 365 operated by 21Vianet is not supported）.
```
Comment (JohanFreelancer9) [inline Teams/room-systems/requirements.md]:
```suggestion
> Microsoft Teams Rooms is intended for use with Microsoft Teams, Skype for Business Server 2019, Skype for Business Server 2015, or Skype for Business Online（Office 365 operated by 21Vianet is not supported.
```

Comment (scnbwy) [reply Teams/room-systems/requirements.md]: Maybe "(Office365 operated by 21Vianet is not supported Now)" is the best description.
Perhaps in the future the MTR product group will modify the code to support 21Vianet
What do you think?

**Comment by scnbwy** [comment]
> 复制审核 - 请重新插入支架。



> 复制审核 - 请重新插入支架。

@JohanFreelancer9 Sorry, this is the first time I've modified MS Docs on github. If possible, please tell me what else to do.

**Comment by JohanFreelancer9** [comment]
@scnbwy Thank you the copy is fine now.

**Comment by scnbwy** [comment]
> @scnbwy谢谢你的副本现在好了。

So, just wait for the Docs update to be described?

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by jambirk** [comment]
@sohailta please review this and advise.  Not aware of this China 21Vianet account limitation.

**Comment by scnbwy** [comment]
I have create a support ticket #:16065906
Confirmed China 21Vianet does not support

**Comment by scanum** [comment]
@sohailta Hi Sohail, any update on this PR? Thanks!

**Comment by scnbwy** [comment]
> We don't recommend adding a specific provider here. We may be ok to add a statement/ note that "Microsoft Teams Rooms do not yet support Gallatin, GCC or GCC-High environments."

Good

File: Teams/ room-systems/requirements.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ Also, refer to:
 - [License options based on your plan: Microsoft Teams Rooms](/SkypeForBusiness/skype-for-business-and-microsoft-teams-add-on-licensing/license-options-based-on-your-plan/skype-room-systems-v2)
 
 > [!NOTE]
-> Microsoft Teams Rooms is intended for use with Microsoft Teams, Skype for Business Server 2019, Skype for Business Server 2015, or Skype for Business Online.
+> Microsoft Teams Rooms is intended for use with Microsoft Teams, Skype for Business Server 2019, Skype for Business Server 2015, or Skype for Business Online（Office365 operated by 21Vianet is not supported now).
 >
```
Comment (nenonix) [inline Teams/ room-systems/requirements.md]:
```suggestion
> Microsoft Teams Rooms is intended for use with Microsoft Teams, Skype for Business Server 2019, Skype for Business Server 2015, or Skype for Business Online. 
>
> Microsoft Teams Rooms do not support Gallatin, GCC or GCC-High environments.
> 
```

**Comment by scanum** [comment]
@sohailta Could you please review changes? thanks!

File: Teams/ room-systems/requirements.md (hunk 2 above)
Comment (jambirk) [reply Teams/ room-systems/requirements.md]: Gallatin is an internal code name, not for general consumption. @sohailta and I will have a look at this later today when we confer about another issue.

**Comment by jambirk** [comment]
given the conflicts that exist for this PR, I am closing this PR and implementing in a separate PR

**Comment by jambirk** [comment]
the fix was implemented in https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/pull/3185

**Pull request closed without merging**